heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -436,6 +436,10 @@ public static void persistUpdatedPackingPlan(String topologyName,
                                                SchedulerStateManagerAdaptor stateManager) {
     LOG.log(Level.INFO, "Updating scheduled-resource in packing plan: {0}", topologyName);
     PackingPlanProtoSerializer serializer = new PackingPlanProtoSerializer();
+    if (stateManager.getPackingPlan(topologyName) != null) {
+      stateManager.deletePackingPlan(topologyName);
```

Comment:
What do you think about reusing this method in `UpdateTopologyManager.updateTopology Line#96`? Or even adding a new `updatePacking` method to `SchedulerStateManagerAdaptor` to avoid confusion?